packages/graph/Makefile.toml

```
@@ -0,0 +1,46 @@
+extend = { path = "../../.github/scripts/rust/Makefile.toml" }
+
+[tasks.crate-volume]
+private = true
+command = "docker"
+args = ["volume", "create", "--name", "hash-graph-data"]
+
+[tasks.touch-environment]
+private = true
+command = "touch"
+args = ["${CARGO_MAKE_CURRENT_TASK_INITIAL_MAKEFILE_DIRECTORY}/.env.local"]
+
+[tasks.yarn]
+category = "Deploy"
+description = "Installs the required dependencies"
+command = "yarn"
+cwd = "${CARGO_MAKE_CURRENT_TASK_INITIAL_MAKEFILE_DIRECTORY}"
```

Does this need to run from cwd `${CARGO..}/migrations`? It might work now because it will go to the monorepo and install everything anyway
```suggestion
cwd = "${CARGO_MAKE_CURRENT_TASK_INITIAL_MAKEFILE_DIRECTORY}/migrations"
```